<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pandemonium_mainwindow</class>
 <widget class="QMainWindow" name="pandemonium_mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pandemonium</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset resource="../Icons/icons.qrc">
         <normaloff>:/nuvola/configure.png</normaloff>:/nuvola/configure.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>&amp;Kernel</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>&amp;Kernel</string>
               </property>
               <property name="buddy">
                <cstring>kernel_path</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="kernel_path"/>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="select_kernel_path">
               <property name="text">
                <string>Select</string>
               </property>
               <property name="icon">
                <iconset resource="../Icons/icons.qrc">
                 <normaloff>:/nuvola/fileopen.png</normaloff>:/nuvola/fileopen.png</iconset>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Kernel Process ID</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="activate_kernel">
               <property name="text">
                <string>Activate</string>
               </property>
               <property name="icon">
                <iconset resource="../Icons/icons.qrc">
                 <normaloff>:/nuvola/run.png</normaloff>:/nuvola/run.png</iconset>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QPushButton" name="deactivate_kernel">
               <property name="toolTip">
                <string>If pressed, kernel monitoring will be disabled.</string>
               </property>
               <property name="text">
                <string>Deactivate</string>
               </property>
               <property name="icon">
                <iconset resource="../Icons/icons.qrc">
                 <normaloff>:/nuvola/stop.png</normaloff>:/nuvola/stop.png</iconset>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLineEdit" name="kernel_pid">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>&amp;Load Interval</string>
               </property>
               <property name="buddy">
                <cstring>kernel_load_interval</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="kernel_load_interval">
               <property name="minimum">
                <double>0.500000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="monitor_kernel">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Silently monitor the kernel. If the kernel is not active, activate it. Please note that deactivating the kernel will disable this option.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>&amp;Monitor Kernel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="proxy_information">
          <property name="title">
           <string>&amp;Proxy</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QComboBox" name="proxy_type">
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
               <item>
                <property name="text">
                 <string>HTTP</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SOCKS5</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>&amp;Address</string>
               </property>
               <property name="buddy">
                <cstring>proxy_address</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="proxy_address"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>&amp;Port</string>
               </property>
               <property name="buddy">
                <cstring>proxy_port</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&amp;User</string>
               </property>
               <property name="buddy">
                <cstring>proxy_user</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="proxy_user"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>&amp;Password</string>
               </property>
               <property name="buddy">
                <cstring>proxy_password</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="proxy_password">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QSpinBox" name="proxy_port">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>65535</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="save_proxy_information">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="search_urls">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Meta-Data Only</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Paused</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Search Depth</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>URL</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>url_hash (hidden)</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="periodically_list_search_urls">
            <property name="text">
             <string>&amp;Periodically Refresh Table</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="add_search_url">
            <property name="text">
             <string>New URL</string>
            </property>
            <property name="icon">
             <iconset resource="../Icons/icons.qrc">
              <normaloff>:/nuvola/bookmark_add.png</normaloff>:/nuvola/bookmark_add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="list_search_urls">
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="../Icons/icons.qrc">
              <normaloff>:/nuvola/reload.png</normaloff>:/nuvola/reload.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_search_urls">
            <property name="text">
             <string>Remove Selected</string>
            </property>
            <property name="icon">
             <iconset resource="../Icons/icons.qrc">
              <normaloff>:/nuvola/editdelete.png</normaloff>:/nuvola/editdelete.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset resource="../Icons/icons.qrc">
         <normaloff>:/nuvola/www.png</normaloff>:/nuvola/www.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Discovered &amp;URLs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="periodically_list_parsed_urls">
            <property name="text">
             <string>&amp;Periodically Refresh Table</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="purge_unvisited_visited_urls">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove pandemonium_visited_urls.db. The parsed URLs will not be removed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Purge</string>
            </property>
            <property name="icon">
             <iconset resource="../Icons/icons.qrc">
              <normaloff>:/nuvola/trashcan_empty.png</normaloff>:/nuvola/trashcan_empty.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toggle_all">
            <property name="toolTip">
             <string>If pressed, the periodic populating of the table will be disabled.</string>
            </property>
            <property name="text">
             <string>Toggle All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toggle_none">
            <property name="toolTip">
             <string>If pressed, the periodic populating of the table will be disabled.</string>
            </property>
            <property name="text">
             <string>Toggle None</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="parsed_urls">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Export</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>URL</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>&amp;Limit</string>
            </property>
            <property name="buddy">
             <cstring>page_limit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="page_limit">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>1000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2500</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7500</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10000</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="page">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>Page 1</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="export_toggled">
            <property name="text">
             <string>Export</string>
            </property>
            <property name="icon">
             <iconset resource="../Icons/icons.qrc">
              <normaloff>:/nuvola/fileexport.png</normaloff>:/nuvola/fileexport.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="list_parsed_urls">
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="../Icons/icons.qrc">
              <normaloff>:/nuvola/reload.png</normaloff>:/nuvola/reload.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_all_parsed_urls">
            <property name="text">
             <string>Remove All</string>
            </property>
            <property name="icon">
             <iconset resource="../Icons/icons.qrc">
              <normaloff>:/nuvola/editdelete.png</normaloff>:/nuvola/editdelete.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_selected_parsed_urls">
            <property name="text">
             <string>Remove Selected</string>
            </property>
            <property name="icon">
             <iconset resource="../Icons/icons.qrc">
              <normaloff>:/nuvola/editdelete.png</normaloff>:/nuvola/editdelete.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Export_Definition"/>
    <addaction name="action_Statistics_Window"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="../Icons/icons.qrc">
     <normaloff>:/nuvola/exit.png</normaloff>:/nuvola/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Statistics_Window">
   <property name="text">
    <string>&amp;Statistics Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Export_Definition">
   <property name="text">
    <string>&amp;Export Definition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tab_widget</tabstop>
  <tabstop>kernel_path</tabstop>
  <tabstop>select_kernel_path</tabstop>
  <tabstop>kernel_pid</tabstop>
  <tabstop>activate_kernel</tabstop>
  <tabstop>deactivate_kernel</tabstop>
  <tabstop>kernel_load_interval</tabstop>
  <tabstop>monitor_kernel</tabstop>
  <tabstop>proxy_information</tabstop>
  <tabstop>proxy_type</tabstop>
  <tabstop>proxy_address</tabstop>
  <tabstop>proxy_port</tabstop>
  <tabstop>proxy_user</tabstop>
  <tabstop>proxy_password</tabstop>
  <tabstop>save_proxy_information</tabstop>
  <tabstop>search_urls</tabstop>
  <tabstop>periodically_list_search_urls</tabstop>
  <tabstop>add_search_url</tabstop>
  <tabstop>list_search_urls</tabstop>
  <tabstop>remove_search_urls</tabstop>
  <tabstop>periodically_list_parsed_urls</tabstop>
  <tabstop>purge_unvisited_visited_urls</tabstop>
  <tabstop>toggle_all</tabstop>
  <tabstop>toggle_none</tabstop>
  <tabstop>parsed_urls</tabstop>
  <tabstop>page_limit</tabstop>
  <tabstop>page</tabstop>
  <tabstop>export_toggled</tabstop>
  <tabstop>list_parsed_urls</tabstop>
  <tabstop>remove_all_parsed_urls</tabstop>
  <tabstop>remove_selected_parsed_urls</tabstop>
 </tabstops>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
